client/src/ui/molecules/language-toggle/index.scss

```diff
@@ -4,11 +4,21 @@
 
 ul.language-toggle {
   display: flex;
-  justify-content: flex-end;
-  padding: $base-spacing;
+  text-align: right;
```

Comment:
Logical property would be `text-align: end` (or is tooling taking care of this?)